advanced/scipy_sparse/examples/lobpcg_sakurai.py

```diff
@@ -41,13 +41,13 @@ def sakurai(n):
 #
 n = 2500
 A, B, w_ex = sakurai(n)  # Mikota pair
-X = np.rand(n, m)
+X = np.random.rand(n, m)
```

Comment:
```suggestion
X = np.random.random((n, m))
```

Should move this over to the new rng structure eventually:

```
rng = np.default_rng()
X = rng.random((n, m))
```